Source/SelfService/Web/application/create.tsx

```diff
@@ -3,140 +3,154 @@
+                color: themeDark.palette.error.dark,
+            },
         },
-        '&.MuiStepIcon-root.Mui-active .MuiStepIcon-text': {
-            fill: '#B3BBFB'
+        '& .MuiOutlinedInput-root.Mui-error .MuiOutlinedInput-notchedOutline': {
+            color: themeDark.palette.error.dark,
+            borderColor: themeDark.palette.error.dark
         },
-        '&.MuiStepIcon-root .MuiStepIcon-text': {
-            fill: '#93959F'
+        '& .MuiFormHelperText-root.Mui-error': {
+            color: themeDark.palette.error.light,
+            letterSpacing: '0.4px'
         }
+    },
+    actionBtnWrapper: {
+        [themeDark.breakpoints.down('sm')]: {
+            marginBlockStart: '60px'
+        },
+    },
+    actionButtons: {
+        color: themeDark.palette.text.primary,
+        fontSize: '0.875rem',
+        letterSpacing: '0.06em'
+    },
+    alertWrapper: {
+        'marginBlockStart': '48px',
+        'borderColor': themeDark.palette.error.dark,
+        'textAlign': 'left',
+        '& .MuiAlert-icon': {
+            color: themeDark.palette.error.dark,
+        }
+    },
+    supportLink: {
+        color: themeDark.palette.primary.main,
+        textDecoration: 'underline',
+        cursor: 'pointer',
+    }
+};
+
+interface ErrorStates {
+    applicationNameError: {
+        message: String,
+        isError: boolean,
+    },
+    contactNameError: {
+        message: String,
+        isError: boolean,
+    },
+    contactEmailError: {
+        message: String,
+        isError: boolean,
     }
 };
 
+const initialErrorState: ErrorStates = {
+    applicationNameError: {
+        message: '',
+        isError: false
+    },
+    contactNameError: {
+        message: '',
+        isError: false
+    },
+    contactEmailError: {
+        message: '',
+        isError: false
+    },
+};
+
+const initialEnvironmentState = [
+    {
+        name: 'Production*',
+        shortName: 'Prod',
+        checked: true,
+        disabled: true,
+        customerTenants: [Guid.create().toString()] as string[]
+    },
+    {
+        name: 'Development',
+        shortName: 'Dev',
+        checked: false,
+        disabled: false,
+        customerTenants: [Guid.create().toString()] as string[]
+    },
+    {
+        name: 'Test',
+        shortName: 'Test',
+        checked: false,
+        disabled: false,
+        customerTenants: [Guid.create().toString()] as string[]
+    },
+];
 
 export const Create: React.FunctionComponent<Props> = (props) => {
     const history = useHistory();
     const { enqueueSnackbar } = useSnackbar();
-    const newApplicationId = Guid.create().toString();
-    const steps = [
-        'Provide a name',
-        'Provide contact info',
-        'Setup Environments'
-    ];
-
 
-    const [activeStep, setActiveStep] = React.useState(0);
+    const newApplicationId = Guid.create().toString();
     const [application, setApplication] = React.useState({
```

Comment:
instead of writing `React.useState` multiple times, perhaps we could do `import { React, useState } from "react"` in line 4?